esignet-default.properties

```diff
@@ -96,11 +96,11 @@ mosip.esignet.kafka.linked-auth-code.topic=esignet-consented
 
 mosip.esignet.integration.scan-base-package=io.mosip.authentication.esignet.integration,io.mosip.esignet.mock.integration
 mosip.esignet.integration.binding-validator=BindingValidatorServiceImpl
-mosip.esignet.integration.authenticator=IdaAuthenticatorImpl
+mosip.esignet.integration.authenticator=SunbirdRCAuthenticationService
 mosip.esignet.integration.key-binder=IdaKeyBinderImpl
 mosip.esignet.integration.audit-plugin=IdaAuditPluginImpl
 mosip.esignet.integration.captcha-validator=GoogleRecaptchaValidatorService
-mosip.esignet.integration.vci-plugin=IdaVCIssuancePluginImpl
+mosip.esignet.integration.vci-plugin=SunbirdRCVCIssuancePlugin
```

Comment:
Undo the above 2 changes, we want sunbird RC to be used only in mock eSignet

Reply:
this  is not taken care. We should have IDA in eSignet-default.properties and Sunbird in eSignet-mock.properties.

Reply:
done the changes